src/config/config.go

```diff
@@ -89,6 +89,13 @@ type User struct {
 	Password string `valid:"required"`
 }
 
+type PrivacyDashboard struct {
+	HostName  string
+	Version   string
+	Status    int
+	StatusStr string
```

Comment:
Remove `Status` and `StatusStr` fields. If `HostName` and `Version` is present, it is already understood the admin dashboard is deployed.